components/browser/search/src/main/java/mozilla/components/browser/search/SearchEngineManager.kt

```diff
@@ -39,26 +39,35 @@ class SearchEngineManager(
     var defaultSearchEngine: SearchEngine? = null
 
     /**
-     * Asynchronously load search engines from providers. Inherits caller's [CoroutineScope.coroutineContext].
+     * Asynchronously load search engines from providers. Inherits caller's [CoroutineContext].
      */
     @Synchronized
-    suspend fun load(context: Context): Deferred<SearchEngineList> = coroutineScope {
+    suspend fun loadAsync(context: Context): Deferred<SearchEngineList> = coroutineScope {
```

Comment:
This is a breaking API change that would affect consuming apps, we might want to just suppress this warning instead.

Reply:
Nice! I didn't catch that. 👏